### PR TITLE
FTUE: Quick Tip Skeleton

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Part of presentational elements for getting to an animated skeleton of the help center.

## Summary
Adds base layout for the quick tips & adds some copy.

## Relevant Technical Choices
NA

## To-do
Store the autoplaying looping videos somewhere and add them to the quick tips. just using a placeholder for now

## User-facing changes
NA

## Testing Instructions
Check out the storybook implementation at stories editor -> components -> Help Center -> Quick Tips
https://user-images.githubusercontent.com/35983235/106304714-f12e1b80-6218-11eb-80d9-80f6bb520f83.mp4

### QA

- [x] This is a non-user-facing change and requires no QA

### UAT

- [x] UAT should use the same steps as above.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [ ] I have added a matching `Type: XYZ` label to the PR

---

Partially addresses #5886 
Partially addresses #5884

### CODE REVIEW

File: assets/src/edit-story/components/helpCenter/constants.js [hunk 1]
```diff
@@ -36,30 +36,81 @@ import { __ } from '@wordpress/i18n';
...
+        'web-stories'
+      ),
+    ],
   },
   addLinks: {
     title: __('Add links to design elements', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Go to the Design tab. Scroll down to Links and enter a URL. To remove a link, select the “X” to the right of URL box.',
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/helpCenter/constants.js]: it RTL would this be on the _left_ of the URL box?

Comment (maxyinger) [reply assets/src/edit-story/components/helpCenter/constants.js]: most likely. Do we have any translation fns where we could specify that it should be left in RTL?

@swissspidy

**Review by BrittanyIRL — APPROVED** [review]
LGTM!

**Review by embarks — APPROVED** [review]

File: assets/src/edit-story/components/helpCenter/constants.js (hunk 1 above)
Comment (swissspidy) [reply assets/src/edit-story/components/helpCenter/constants.js]: You could do 

```js
const { cdnURL, isRTL } = useConfig();

...

description: isRTL ? __('...') : __('...')`
```

But generally speaking I would highly recommend not using any such directions in texts. For i18n and a11y reasons. "Scroll down" and "to the right of" doesn't say much for screen reader users.

I'd use something like

```
Go to the Links section under the Design tab and enter a URL. To remove a link, select the "X" button.
```

Heck, even this isn't great because "X" doesn't say much and the actual button is hopefully announced differently to screen readers.

So maybe something like this:

```
Go to the Links section under the Design tab and enter a URL. To remove a link, select the "X" (Clear) button.
```

Could even make the `(Clear)` only visible  to screen readers by using `TranslateWithMarkup`

Comment (BrittanyIRL) [reply assets/src/edit-story/components/helpCenter/constants.js]: Let's clear this with design, but i think @swissspidy brings up a great point.

File: package-lock.json [hunk 2]
```diff
@@ -14413,12 +14413,19 @@
       }
     },
     "dom-helpers": {
-      "version": "5.1.4",
-      "resolved": "https://registry.npmjs.org/dom-helpers/-/dom-helpers-5.1.4.tgz",
-      "integrity": "sha512-TjMyeVUvNEnOnhzs6uAn9Ya47GmMo3qq7m+Lr/3ON0Rs5kHvb8I+SQYjLUSYn7qhEm0QjW0yrBkvz9yOrwwz1A==",
+      "version": "5.2.0",
```
Comment (spacedmonkey) [inline package-lock.json]: Are these changes needed?

File: assets/src/edit-story/components/helpCenter/constants.js (hunk 1 above)
Comment (BrittanyIRL) [reply assets/src/edit-story/components/helpCenter/constants.js]: Design agrees that the above is a good thing to implement, thanks @swissspidy ! 

@littlemilkstudio Let's update to use: Go to the Links section under the Design tab and enter a URL. To remove a link, select the "X" (Clear) button.

where the (Clear) shows up for just screen readers.
